packages/cw-orch-cli-derive/src/lib.rs

```diff
@@ -3,17 +3,17 @@ extern crate proc_macro;
 use quote::ToTokens;
 use syn::{parse_macro_input, parse_quote, DeriveInput};
 
-// TODO: check what's the best way to serialize for user
 #[proc_macro_attribute]
 pub fn cli(_attr: TokenStream, input: TokenStream) -> TokenStream {
     let input = parse_macro_input!(input as DeriveInput);
     let output: DeriveInput = match input.data {
         syn::Data::Struct(_) => parse_quote!(#input),
         syn::Data::Enum(_) => {
             parse_quote!(
-                #[cfg_attr(feature = "interface", derive(
-                    ::cw_orch_cli::strum::EnumVariantNames,
-                ))]
+                #[cfg_attr(
+                    not(target_arch = "wasm32"),
```

Comment:
I was saying that this line is useless, because you already hide the cli macro behind an interface macro

Reply:
It was not before I moved it to derive